src/plugins/wb-focus/focus.js

```diff
@@ -86,7 +86,7 @@ $document.on( clickEvents, linkSelector, function( event ) {
 	var testHref = event.currentTarget.getAttribute( "href" );
 
 	// Same page links only
-	if ( testHref.charAt( 0 ) === "#" && !event.isDefaultPrevented() &&
+	if ( wb.jqEscape( testHref.substring( 1 ) ).length !== 0 && testHref.charAt( 0 ) === "#" && !event.isDefaultPrevented() &&
```

Comment:
Did you tried to only test if "testHref" is longer than 1 character instead of calling the wb.jqEscape ?
Like: 
```suggestion
	if ( testHref.length > 1 && testHref.charAt( 0 ) === "#" && !event.isDefaultPrevented() &&
```